src/parser/CMakeLists.txt

```diff
@@ -110,7 +110,7 @@ set_source_files_properties(${CMAKE_CURRENT_BINARY_DIR}/smt2/smt2_lexer.cpp PROP
 # The FlexLexer.h header is installed to /usr/include/, but is not found by the
 # cross-compilation environment. As a workaround we'll copy the header to the
 # current build directory and set the FLEX_INCLUDE_DIRS variable accordingly.
-if(WIN32)
+if(WIN32 AND NOT ("${CMAKE_GENERATOR}" STREQUAL "MSYS Makefiles"))
```

Comment:
Should we maybe have a flag `CROSS_COMPILE_WIN` (or something similar) that indicates whether we cross-compile for windows or not?

Reply:
Let's merge this fix now and cleanup do the cleanup in a separate PR.